Add no seed installation manual (#9349)
decidim/decidim: docs/modules/install/pages/manual.adoc

bin/rails db:seed
----

> [NOTE]

Reviewer: I've just realized that we should change the initial phrase of "We should now create your database:" (line 116). How about "We should now create your database. For a first local installation, we recommend to start with some example contents (also known as seeds)."

====

> Please refer to xref:install:noseed.adoc[No-seed Installation] section if you want to setup your instance with no seeds

Reviewer: I think seeds is too technical/rails specific and probably something that isn't familiar with programming wouldn't know what's that. How about renaming this to "empty installation" or something like that instead of "no-seed installation" ? 

```suggestion
Please refer to xref:install:empty-database.adoc[Empty database installation] section if you want to setup your instance on an empty database (without any seeds)
```

Author: Good point. I was thinking of the same issue.
